captum/attr/_core/grad_cam.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import torch
+
+from .._utils.attribution import LayerAttribution
+from .._utils.common import format_input, _format_additional_forward_args
+from .._utils.gradient import compute_layer_gradients_and_eval
+
+
+class LayerGradCam(LayerAttribution):
+    def __init__(self, forward_func, layer, device_ids=None):
```

Comment:
One thing I remembered is that this layer ones need probably GPU tests ...

Reply:
Good point, yeah it would be good to add some!